### DISCUSSION TITLE
Combine some of the global emotes that have aliases

### DISCUSSION THREAD

**Discussion by mmattbtw** [Chatterino/chatterino2#3435] [Ideas]
In the Chatterino emote picker, it shows the emote for every alias. That makes it so there are duplicates in the list. So I'm thinking that you can combine the emotes with aliases (ex: :p, :-p, :P, ;-p, ;-P, ;p, ;P) into one emote. (maybe even showing the aliases on the popup)

![Image of the Chatterino emote picker, showing all of the smiley global Twitch emotes, in which some are duplicates.](https://user-images.githubusercontent.com/30363562/147530914-29a1c85e-04d5-4286-bfca-94b85a5635a2.png)

**Comment by Mm2PL** [comment]
They are different emotes according to the API. Some emotes like the smiles used to have regex names but that got changed a while ago.
